ceno_zkvm/src/scheme/prover.rs

```diff
@@ -100,16 +102,18 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMProver<E, PCS> {
                         PCS::batch_commit_and_write(&self.pk.pp, &witness, &mut transcript)
                             .map_err(ZKVMError::PCSError)?,
                     );
-                    tracing::info!(
-                        "commit to {} traces took {:?}",
-                        circuit_name,
-                        commit_dur.elapsed()
-                    );
+                    // tracing::info!(
```

Comment:
When the PR is ready, please remove commented out code, we can always get it back from the git history.

(Commented out code is fine for a draft.)